content/tutorials/rust/hello-world.md

```diff
@@ -22,274 +24,127 @@ Before starting on this tutorial, you'll need to have completed the following
 [Rust programming language]: https://rustup.rs
 [getting started]: /docs/getting-started
 
-## Build a simple data streaming App
+## Creating a new Cargo project
 
-This section provides a step-by-step on how to build a simple data streaming app using Rust. If you'd like to download the demo app instead, skip ahead to [Download Rust Demo Apps](#download-rust-demo-apps).
+We'll be using Rust's package manager [Cargo] to set up our project. Cargo
+is helpful because it manages compiling our code as well as downloading
+dependencies that we need.
 
-#### Implement Producer/Consumer exchange
+[Cargo]: https://doc.rust-lang.org/book/ch01-03-hello-cargo.html
 
-Fluvio client needs a [Profile](/docs/cli/profiles) to identify the location and the authorization token of the cluster. The file was generated during cluster setup and it is available for download in your <a href="https://app.fluvio.io" target="_blank">Fluvio Cloud</a> account.
-
-#### Create Producer Package
-
-Use <a href="https://doc.rust-lang.org/cargo/" target="_blank">Cargo</a> _--bin_ flag to create an application:
-
-```bash
-$ cargo new fluvio-rust-produce --bin
-Created binary (application) `fluvio-rust-produce` package
-```
-
-This results in following directory layout:
+To create a new Rust project with cargo, run
 
 ```bash
-$ cd fluvio-rust-produce/
-$ tree
-.
-├── Cargo.toml
-└── src
-    └── main.rs
+$ cargo new --bin hello-fluvio
+     Created binary (application) `hello-fluvio` package
 ```
 
-Update _Cargo.toml_ file to add dependencies:
+One of the first things we need to do is add our dependencies. We're going to need
+the `fluvio` crate as well as `async-std` because the Fluvio client is async.
+In your project folder, edit your `Cargo.toml` and add lines under your `[dependencies]`
+section to import `fluvio` and `async-std`. It should look something like this when you're done
 
 ```bash
+$ cd hello-fluvio
 $ cat Cargo.toml
 [package]
-name = "fluvio-rust-produce"
+name = "hello-fluvio"
 version = "0.1.0"
-authors = ["user <user@fluvio.io>"]
+authors = ["Your name <your_email@example.com>"]
 edition = "2018"
 
 [dependencies]
-log = "0.4.8"
-futures = { version = "0.3.4", features = ['async-await'] }
-flv-future-aio = { version = "1.0.0" }
-flv-client = { version = "1.0.0"}
-kf-protocol = { version = "1.0.0 "}
+fluvio = "0.1.0"
+async-std = "1.0.0"
 ```
 
-
-##### Producer Code
-
-Add the following code in  _src/main.rs_ file:
+Now let's head on over to the `src/main.rs` file. This is where your `main` function
```

Comment:
Since we’ll be using async code, let’s set up a helper function for main that will handle all of our async needs.